precompile/contracts/rewardmanager/contract.go

```diff
@@ -21,14 +21,17 @@ import (
 )
 
 const (
-	DuplicatedLogGas      uint64 = 375 // TODO: duplicated from params/protocol_params.go to avoid import cycle in tests.
-	DuplicatedLogTopicGas uint64 = 375 // TODO: duplicated from params/protocol_params.go to avoid import cycle in tests.
+	AllowFeeRecipientsGasCost         uint64 = contract.WriteGasCostPerSlot + allowlist.ReadAllowListGasCost // write 1 slot + read allow list
+	UpgradedAllowFeeRecipientsGasCost uint64 = contract.LogTopicGas + contract.WriteGasCostPerSlot + allowlist.ReadAllowListGasCost
```

Comment:
I think `PreDUpgrade...` and `PostDUpgrade...` prefixes were good (unless we name it `PreDFork` etc)

Reply:
Done